test/unit/core/connect.test.js

```diff
@@ -107,7 +107,7 @@ describe('Connect Tests', function() {
 
     connect({ host: '240.0.0.1' }, cancellationToken, (err, conn) => {
       expect(err).to.exist;
-      expect(err).to.match(/connection establishment was cancelled/);
+      expect(err).to.match(/(connection establishment was cancelled|connect ENETUNREACH)/);
```

Comment:
This also looks like a bug to me. The test here is validating that a cancellation token was used to cancel an in-progress connection attempt. It looks like instead you are getting a connection failure. The test itself is brittle, it uses an IP address which should black hole all requests but maybe it's not possible on windows. I think we should skip this for now, and file a ticket to change the test to not use this hacky IP address

Reply:
[NODE-2941](https://jira.mongodb.org/browse/NODE-2941)